README.md

```diff
@@ -15,21 +15,21 @@
 
 ### Deployment
 
-To use the ExtendedDaemonSet controller in your Kubernetes cluster, deploy its resources definitions in the `/deploy` folder.
+To use the ExtendedDaemonSet controller in your Kubernetes cluster, only two commands are required:
 
 First, deploy the Custom Resources Definitions:
 
 ```console
-$ kubectl apply -f ./deploy/crds
+$ make install
 ```
 
-If you want to deploy a specific controller version, the version can be set in the `deploy/operator.yaml` file. By default, the `latest` docker image tag is used.
+Then deploy default manifest (uses Kustomize)
```

Comment:
should we add Kustomize cli as project dependency? are we using only kustomize included in the `kustomize apply -k` option?

Reply:
Yes, the generated Makefile deals with that automatically